src/Tags/Concerns/QueriesConditions.php

```diff
@@ -45,6 +45,8 @@ protected function queryCondition($query, $field, $condition, $value)
     {
         $regexOperator = $value ? 'regexp' : 'not regexp';
 
+        $field = str_replace('.', '->', $field);
```

Comment:
Would this not introduce a breaking change for those who are currently using the `:date_field->end:is_after="today"` sytax?

Reply:
Ah nevermind, I was reading it as `'->', '.'` for some reason 🤦‍♂️